editor.planx.uk/src/components/Feedback/index.tsx

```diff
@@ -250,6 +251,12 @@ const Feedback: React.FC = () => {
                 label="Comment"
                 showArrow
               />
+              <FeedbackOption
+                onClick={() => handleFeedbackViewClick("inaccuracy")}
+                Icon={RuleIcon}
+                label="Inaccuracy"
+                showArrow
+              />
```

Comment:
We could conditionally display this if the node type matches the previous places we used to show this option, but that seems a little inconsistent for users potentially? I've opted against it here to very much open to thoughts.

Reply:
I think this is right - easy to imagine someone wanting to report "inaccurate" help text, etc :+1:

Reply:
Yep - agreed!